apps/yggdrasil_web/web/controllers/character_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule YggdrasilWeb.CharacterController do
+  use YggdrasilWeb.Web, :controller
+  use Guardian.Phoenix.Controller
+
+  alias Yggdrasil.Character
+
+  def index(conn, _params, user, _claims) do
+    user_id = user.id
```

Comment:
Would it be better to add a permission system and have the API accept the user id? Seems like a better API would be to include the user id in the url anyway, isn't that more conventional?

```
/characters # all characters I can see
/characters?userid=12 # characters only for user 12
```